lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -170,6 +172,15 @@ local function submitFormForPlayer( data, endpoint, ply )
     )
 
     recordPlayerSubmission( ply )
+
+    local staffRanks = { moderator = true }
+    for _,p in ipairs( player.GetHumans() ) do
```

Comment:
NIT: its also common to name the player variable in a loop like this `ply` instead of a single letter variable. Its not a big deal in a small loop like this though.

Reply:
I used `p` to represent "player" because `ply` was already used in that code block. Would you like me to use `Ply` so that it still reads as `ply`, but just has a capital P?

Reply:
using `Ply` and `ply` in the same spot is going to lead into confusion in the future, what about renaming the ply to `formSubmitter` or the loop p to something like `receivingPlayer`

Reply:
Renamed original `ply` to `formSubmitter` then renamed `p` to `ply`.